### ISSUE TITLE
Expose `set_dsp_mcacc_memory_set` AVR command as a service

### ISSUE THREAD

**Issue by crowbarz** [crowbarz/ha-pioneer_async#42]
Follow-up from crowbarz/aiopioneer#33

**Comment by crowbarz** [comment]
Possibly also add a select entity that renders a drop-down to select the MCACC memory.

**Comment by crowbarz** [comment]
@Tamas-Toth-ebola as a first step, you can use action `pioneer_async.set_dsp_settings` to set the active MCACC memory set. I think this has been possible for a while, though in HA integration version 0.11 the available DSP options are now selectable in the Developer Tools > Actions UI.

The next step will be to add that select entity previously mentioned to both show the currently active MCACC memory set, as well as be able to change it from a HA dashboard. This is still on the todo list, though recent changes to `aiopioneer` have made its implementation much more straightforward.

**Comment by Tamas-Toth-ebola** [comment]
Yes and very lot of thanks for your efforts. Previously I used the send command method, what you mentioned before, but with your latest HA integration, I had already switched to the DSP settings method, which is working fine.

**Comment by crowbarz** [comment]
0.12 added a number entity for MCACC memory set, so you should be able to set that entity to change the selected memory set. You can also now add that entity directly to a dashboard so it can be changed via the UI. So I think this feature request is now completed.

There are additional plans to allow configuration of the selected MCACC memory set, but that will be a fairly low priority.

**Comment by Tamas-Toth-ebola** [comment]
Tested, works, nice, thanks 😉
